vars/util.groovy

```diff
@@ -405,8 +405,8 @@ def retrieveFailedTests(String buildUrl = "${BUILD_URL}") {
                                     .replaceAll('-', '_')
                 failedTest.url = "${buildUrl}testReport/${packageName}/${className}/${urlLeaf}"
 
-                failedTest.details = testCase.errorDetails
-                failedTest.stacktrace = testCase.errorStackTrace
+                failedTest.details = testCase.errorDetails == 'null' ? '' : testCase.errorDetails
```

Comment:
I would prefer something like `[null, "null"].contains(testCase.errorDetails) ? '' : testCase.errorDetails`

Reply:
if this is safer, gonna create a small PR for that :)

Reply:
https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/185